<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<ns2:test-suite xmlns:ns2="urn:model.allure.qatools.yandex.ru" start="1419241296590" stop="1419241719951">
    <name>TestRun</name>
    <test-cases>
        <test-case start="1419241333424" stop="1419241339714" status="passed">
            <name>userEmptyLogin[[Ivan, Ivanov, Incorrect login or password]]</name>
            <steps>
                <step start="1419241337072" stop="1419241339565" status="passed">
                    <name>login</name>
                    <title>When I login as   and Ivanov</title>
                    <attachments/>
                    <steps/>
                </step>
                <step start="1419241339565" stop="1419241339648" status="passed">
                    <name>incorrectLoginPassword</name>
                    <title>Verify Incorrect Login or Password message</title>
                    <attachments/>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <labels>
                <label name="feature" value="Login"/>
            </labels>
        </test-case>
        <test-case start="1419241340047" stop="1419241340880" status="passed">
            <name>userEmptyPassword[[Ivan, Ivanov, Incorrect login or password]]</name>
            <steps>
                <step start="1419241340200" stop="1419241340823" status="passed">
                    <name>login</name>
                    <title>When I login as Ivan and  </title>
                    <attachments/>
                    <steps/>
                </step>
                <step start="1419241340823" stop="1419241340880" status="passed">
                    <name>incorrectLoginPassword</name>
                    <title>Verify Incorrect Login or Password message</title>
                    <attachments/>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <labels>
                <label name="feature" value="Login"/>
            </labels>
        </test-case>
        <test-case start="1419241340962" stop="1419241341761" status="passed">
            <name>userIncorrectLogin[[Admin_Admin, 111.0, Incorrect login or password]]</name>
            <steps>
                <step start="1419241341164" stop="1419241341711" status="passed">
                    <name>login</name>
                    <title>When I login as Admin_Admin and 111.0</title>
                    <attachments/>
                    <steps/>
                </step>
                <step start="1419241341711" stop="1419241341760" status="passed">
                    <name>incorrectLoginPassword</name>
                    <title>Verify Incorrect Login or Password message</title>
                    <attachments/>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <labels>
                <label name="feature" value="Login"/>
            </labels>
        </test-case>
        <test-case start="1419241341830" stop="1419241345224" status="passed">
            <name>userLogin[[Admin_Admin, Admin111, Admin Admin (admin)]]</name>
            <steps>
                <step start="1419241341994" stop="1419241344725" status="passed">
                    <name>login</name>
                    <title>When I login as Admin_Admin and Admin111</title>
                    <attachments/>
                    <steps/>
                </step>
                <step start="1419241344725" stop="1419241344804" status="passed">
                    <name>loggedUserName</name>
                    <title>Then get logged user name</title>
                    <attachments/>
                    <steps/>
                </step>
                <step start="1419241344804" stop="1419241345223" status="passed">
                    <name>navigateTo</name>
                    <title>When I navigate to Logout</title>
                    <attachments/>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <labels>
                <label name="feature" value="Login"/>
            </labels>
        </test-case>
        <test-case start="1419241345226" stop="1419241346362" status="passed">
            <name>userLogin[[Mentor_Mentor, Mentor111, Mentor Mentor (mentor)]]</name>
            <steps>
                <step start="1419241345379" stop="1419241346048" status="passed">
                    <name>login</name>
                    <title>When I login as Mentor_Mentor and Mentor111</title>
                    <attachments/>
                    <steps/>
                </step>
                <step start="1419241346048" stop="1419241346115" status="passed">
                    <name>loggedUserName</name>
                    <title>Then get logged user name</title>
                    <attachments/>
                    <steps/>
                </step>
                <step start="1419241346115" stop="1419241346361" status="passed">
                    <name>navigateTo</name>
                    <title>When I navigate to Logout</title>
                    <attachments/>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <labels>
                <label name="feature" value="Login"/>
            </labels>
        </test-case>
        <test-case start="1419241346364" stop="1419241347777" status="passed">
            <name>userLogin[[LabManager_LabManager, LabManager111, LabManager LabManager (lab manager)]]</name>
            <steps>
                <step start="1419241346528" stop="1419241347438" status="passed">
                    <name>login</name>
                    <title>When I login as LabManager_LabManager and LabManager111</title>
                    <attachments/>
                    <steps/>
                </step>
                <step start="1419241347438" stop="1419241347484" status="passed">
                    <name>loggedUserName</name>
                    <title>Then get logged user name</title>
                    <attachments/>
                    <steps/>
                </step>
                <step start="1419241347484" stop="1419241347776" status="passed">
                    <name>navigateTo</name>
                    <title>When I navigate to Logout</title>
                    <attachments/>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <labels>
                <label name="feature" value="Login"/>
            </labels>
        </test-case>
        <test-case start="1419241357761" stop="1419241377241" status="passed">
            <name>addUserCustomEmail[[Admin_Admin, Admin111, Ivan, Ivanov, ivanov@epam.com]]</name>
            <steps>
                <step start="1419241357985" stop="1419241361555" status="passed">
                    <name>login</name>
                    <title>When I login as Admin_Admin and Admin111</title>
                    <attachments/>
                    <steps/>
                </step>
                <step start="1419241361555" stop="1419241362480" status="passed">
                    <name>navigateTo</name>
                    <title>When I navigate to Users</title>
                    <attachments/>
                    <steps/>
                </step>
                <step start="1419241362481" stop="1419241366773" status="passed">
                    <name>addNewUserCustomEmail</name>
                    <title>When I set custom email Ivan, Ivanov</title>
                    <attachments/>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <labels>
                <label name="feature" value="Users"/>
            </labels>
        </test-case>
        <test-case start="1419241377298" stop="1419241390438" status="passed">
            <name>adminAddNewUser[[Admin_Admin, Admin111, IvanMentor, IvanovMentor, MENTOR]]</name>
            <steps>
                <step start="1419241377497" stop="1419241378386" status="passed">
                    <name>login</name>
                    <title>When I login as Admin_Admin and Admin111</title>
                    <attachments/>
                    <steps/>
                </step>
                <step start="1419241378386" stop="1419241378882" status="passed">
                    <name>navigateTo</name>
                    <title>When I navigate to Users</title>
                    <attachments/>
                    <steps/>
                </step>
                <step start="1419241378883" stop="1419241380088" status="passed">
                    <name>addUserDifferentRoles</name>
                    <title>When I create users with different roles IvanMentor, IvanovMentor</title>
                    <attachments/>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <labels>
                <label name="feature" value="Users"/>
            </labels>
        </test-case>
        <test-case start="1419241390439" stop="1419241403514" status="passed">
            <name>adminAddNewUser[[Admin_Admin, Admin111, ILabAuto, LabAuto, LAB_MANAGER]]</name>
            <steps>
                <step start="1419241390621" stop="1419241391453" status="passed">
                    <name>login</name>
                    <title>When I login as Admin_Admin and Admin111</title>
                    <attachments/>
                    <steps/>
                </step>
                <step start="1419241391453" stop="1419241391961" status="passed">
                    <name>navigateTo</name>
                    <title>When I navigate to Users</title>
                    <attachments/>
                    <steps/>
                </step>
                <step start="1419241391962" stop="1419241393147" status="passed">
                    <name>addUserDifferentRoles</name>
                    <title>When I create users with different roles ILabAuto, LabAuto</title>
                    <attachments/>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <labels>
                <label name="feature" value="Users"/>
            </labels>
        </test-case>
        <test-case start="1419241403516" stop="1419241416186" status="passed">
            <name>adminAddNewUser[[Admin_Admin, Admin111, Imentee, Imentee, MENTEE]]</name>
            <steps>
                <step start="1419241403728" stop="1419241404481" status="passed">
                    <name>login</name>
                    <title>When I login as Admin_Admin and Admin111</title>
                    <attachments/>
                    <steps/>
                </step>
                <step start="1419241404481" stop="1419241405107" status="passed">
                    <name>navigateTo</name>
                    <title>When I navigate to Users</title>
                    <attachments/>
                    <steps/>
                </step>
                <step start="1419241405107" stop="1419241405883" status="passed">
                    <name>addUserDifferentRoles</name>
                    <title>When I create users with different roles Imentee, Imentee</title>
                    <attachments/>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <labels>
                <label name="feature" value="Users"/>
            </labels>
        </test-case>
        <test-case start="1419241416188" stop="1419241429737" status="passed">
            <name>adminAddNewUser[[Admin_Admin, Admin111, IAdmin, IAdmin, ADMIN]]</name>
            <steps>
                <step start="1419241416342" stop="1419241417636" status="passed">
                    <name>login</name>
                    <title>When I login as Admin_Admin and Admin111</title>
                    <attachments/>
                    <steps/>
                </step>
                <step start="1419241417636" stop="1419241418564" status="passed">
                    <name>navigateTo</name>
                    <title>When I navigate to Users</title>
                    <attachments/>
                    <steps/>
                </step>
                <step start="1419241418564" stop="1419241419419" status="passed">
                    <name>addUserDifferentRoles</name>
                    <title>When I create users with different roles IAdmin, IAdmin</title>
                    <attachments/>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <labels>
                <label name="feature" value="Users"/>
            </labels>
        </test-case>
        <test-case start="1419241429784" stop="1419241432330" status="passed">
            <name>addUserWithoutName[[Admin_Admin, Admin111, Petro, Petrov, You input first name incorrect.]]</name>
            <steps>
                <step start="1419241430021" stop="1419241430706" status="passed">
                    <name>login</name>
                    <title>When I login as Admin_Admin and Admin111</title>
                    <attachments/>
                    <steps/>
                </step>
                <step start="1419241430707" stop="1419241431289" status="passed">
                    <name>navigateTo</name>
                    <title>When I navigate to Users</title>
                    <attachments/>
                    <steps/>
                </step>
                <step start="1419241431289" stop="1419241431996" status="passed">
                    <name>addNewUser</name>
                    <title>When I add new user  ,Petrov</title>
                    <attachments/>
                    <steps/>
                </step>
                <step start="1419241431996" stop="1419241432060" status="passed">
                    <name>emptyNameFieldMessage</name>
                    <title>Verify empty FirstNameField</title>
                    <attachments/>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <labels>
                <label name="feature" value="Users"/>
            </labels>
        </test-case>
        <test-case start="1419241432371" stop="1419241434734" status="passed">
            <name>addUserWithoutSurname[[Admin_Admin, Admin111, Petro, Petrov, You input last name incorrect.]]</name>
            <steps>
                <step start="1419241432602" stop="1419241433219" status="passed">
                    <name>login</name>
                    <title>When I login as Admin_Admin and Admin111</title>
                    <attachments/>
                    <steps/>
                </step>
                <step start="1419241433219" stop="1419241433716" status="passed">
                    <name>navigateTo</name>
                    <title>When I navigate to Users</title>
                    <attachments/>
                    <steps/>
                </step>
                <step start="1419241433716" stop="1419241434411" status="passed">
                    <name>addNewUser</name>
                    <title>When I add new user Admin_Admin, </title>
                    <attachments/>
                    <steps/>
                </step>
                <step start="1419241434411" stop="1419241434481" status="passed">
                    <name>emptySurnameFieldMessage</name>
                    <title>Verify empty LastNameField</title>
                    <attachments/>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <labels>
                <label name="feature" value="Users"/>
            </labels>
        </test-case>
        <test-case start="1419241442793" stop="1419241442872" status="canceled">
            <name>preConditions</name>
            <failure>
                <message>RuntimeException: java.lang.NullPointerException</message>
                <stack-trace>java.lang.RuntimeException: java.lang.NullPointerException
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:161)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:429)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
Caused by: java.lang.NullPointerException
	at khppp.excel.utils.ExcelReader.getCellData(ExcelReader.java:69)
	at khppp.excel.utils.ExcelReader.getRowContains(ExcelReader.java:84)
	at khppp.core.CoreTest.testData(CoreTest.java:57)
	at khppp.tests.BaseCase.testData(BaseCase.java:54)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:135)
	... 23 more
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels/>
        </test-case>
        <test-case start="1419241442872" stop="1419241477402" status="broken">
            <name>addGroupBtnName</name>
            <failure>
                <message>TimeoutException: Timed out after 30 seconds waiting for visibility of element located by By.xpath: //a[contains(.,'Groups')]
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=34.0.5, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webdriver.remote.sessionid=35c93c1e-0cbc-4aa6-8453-e639dbabc83e, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 35c93c1e-0cbc-4aa6-8453-e639dbabc83e</message>
                <stack-trace>org.openqa.selenium.TimeoutException: Timed out after 30 seconds waiting for visibility of element located by By.xpath: //a[contains(.,'Groups')]
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=34.0.5, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webdriver.remote.sessionid=35c93c1e-0cbc-4aa6-8453-e639dbabc83e, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 35c93c1e-0cbc-4aa6-8453-e639dbabc83e
	at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:79)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:228)
	at khppp.application.components.Component.waitFor(Component.java:73)
	at khppp.application.components.Component.waitFor(Component.java:83)
	at khppp.application.components.NavBar.getTab(NavBar.java:23)
	at khppp.application.steps.NavBarSteps.navigateToGroups(NavBarSteps.java:29)
	at khppp.tests.BaseCase.goToGroupsTab(BaseCase.java:67)
	at khppp.tests.GroupsTest.addGroupBtnName(GroupsTest.java:59)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
Caused by: org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"xpath","selector":"//a[contains(.,'Groups')]"}
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.44.0', revision: '76d78cf', time: '2014-10-23 20:02:37'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: driver.version: unknown
Command duration or timeout: 10.50 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=34.0.5, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webdriver.remote.sessionid=35c93c1e-0cbc-4aa6-8453-e639dbabc83e, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 35c93c1e-0cbc-4aa6-8453-e639dbabc83e
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:408)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:204)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:156)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:599)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:352)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:449)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:357)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:344)
	at org.openqa.selenium.support.ui.ExpectedConditions.findElement(ExpectedConditions.java:730)
	at org.openqa.selenium.support.ui.ExpectedConditions.access$000(ExpectedConditions.java:39)
	at org.openqa.selenium.support.ui.ExpectedConditions$4.apply(ExpectedConditions.java:130)
	at org.openqa.selenium.support.ui.ExpectedConditions$4.apply(ExpectedConditions.java:126)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:208)
	... 33 more
Caused by: org.openqa.selenium.remote.ScreenshotException: Screen shot has been taken
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: driver.version: RemoteWebDriver
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:136)
	... 43 more
Caused by: org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"xpath","selector":"//a[contains(.,'Groups')]"}
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.44.0', revision: '76d78cf', time: '2014-10-23 20:02:37'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: driver.version: unknown
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: driver.version: unknown
	at &lt;anonymous class&gt;.FirefoxDriver.prototype.findElementInternal_(file:///C:/Users/YEHOR_~1/AppData/Local/Temp/anonymous9033647809687997824webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:9641:26)
	at &lt;anonymous class&gt;.fxdriver.Timer.prototype.setTimeout/&lt;.notify(file:///C:/Users/YEHOR_~1/AppData/Local/Temp/anonymous9033647809687997824webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:548:5)
</stack-trace>
            </failure>
            <steps>
                <step start="1419241442872" stop="1419241477228" status="broken">
                    <name>navigateToGroups</name>
                    <title>When I navigate to groups</title>
                </step>
            </steps>
            <attachments/>
            <labels>
                <label name="feature" value="Groups"/>
            </labels>
        </test-case>
        <test-case start="1419241477403" stop="1419241496294" status="passed">
            <name>exportBtnName</name>
            <steps>
                <step start="1419241477403" stop="1419241496203" status="passed">
                    <name>navigateToGroups</name>
                    <title>When I navigate to groups</title>
                    <attachments/>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <labels>
                <label name="feature" value="Groups"/>
            </labels>
        </test-case>
        <test-case start="1419241496297" stop="1419241496906" status="passed">
            <name>validGroupsTableHead</name>
            <steps>
                <step start="1419241496297" stop="1419241496709" status="passed">
                    <name>navigateToGroups</name>
                    <title>When I navigate to groups</title>
                    <attachments/>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <labels>
                <label name="feature" value="Groups"/>
            </labels>
        </test-case>
        <test-case start="1419241496908" stop="1419241497392" status="passed">
            <name>validTabName</name>
            <steps>
                <step start="1419241496908" stop="1419241497328" status="passed">
                    <name>navigateToGroups</name>
                    <title>When I navigate to groups</title>
                    <attachments/>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <labels>
                <label name="feature" value="Groups"/>
            </labels>
        </test-case>
        <test-case start="1419241497450" stop="1419241497450" status="canceled">
            <name>addEmptyGroup</name>
            <failure>
                <message>RuntimeException: java.lang.NullPointerException</message>
                <stack-trace>java.lang.RuntimeException: java.lang.NullPointerException
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:161)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:429)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
Caused by: java.lang.NullPointerException
	at khppp.excel.utils.ExcelReader.getCellData(ExcelReader.java:69)
	at khppp.excel.utils.ExcelReader.getRowContains(ExcelReader.java:84)
	at khppp.core.CoreTest.testData(CoreTest.java:57)
	at khppp.tests.BaseCase.testData(BaseCase.java:54)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:135)
	... 23 more
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="feature" value="Groups"/>
            </labels>
        </test-case>
        <test-case start="1419241497492" stop="1419241497492" status="canceled">
            <name>addGroupWithoutDep</name>
            <failure>
                <message>RuntimeException: java.lang.NullPointerException</message>
                <stack-trace>java.lang.RuntimeException: java.lang.NullPointerException
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:161)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:429)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
Caused by: java.lang.NullPointerException
	at khppp.excel.utils.ExcelReader.getCellData(ExcelReader.java:69)
	at khppp.excel.utils.ExcelReader.getRowContains(ExcelReader.java:84)
	at khppp.core.CoreTest.testData(CoreTest.java:57)
	at khppp.tests.BaseCase.testData(BaseCase.java:54)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:135)
	... 23 more
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="feature" value="Groups"/>
            </labels>
        </test-case>
        <test-case start="1419241497539" stop="1419241497539" status="canceled">
            <name>addGroupWithoutName</name>
            <failure>
                <message>RuntimeException: java.lang.NullPointerException</message>
                <stack-trace>java.lang.RuntimeException: java.lang.NullPointerException
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:161)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:429)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
Caused by: java.lang.NullPointerException
	at khppp.excel.utils.ExcelReader.getCellData(ExcelReader.java:69)
	at khppp.excel.utils.ExcelReader.getRowContains(ExcelReader.java:84)
	at khppp.core.CoreTest.testData(CoreTest.java:57)
	at khppp.tests.BaseCase.testData(BaseCase.java:54)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:135)
	... 23 more
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="feature" value="Groups"/>
            </labels>
        </test-case>
        <test-case start="1419241497609" stop="1419241497610" status="canceled">
            <name>groupNameFieldFromSpecialSymbol</name>
            <failure>
                <message>RuntimeException: java.lang.NullPointerException</message>
                <stack-trace>java.lang.RuntimeException: java.lang.NullPointerException
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:161)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:429)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
Caused by: java.lang.NullPointerException
	at khppp.excel.utils.ExcelReader.getCellData(ExcelReader.java:69)
	at khppp.excel.utils.ExcelReader.getRowContains(ExcelReader.java:84)
	at khppp.core.CoreTest.testData(CoreTest.java:57)
	at khppp.tests.BaseCase.testData(BaseCase.java:54)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:135)
	... 23 more
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="feature" value="Groups"/>
            </labels>
        </test-case>
        <test-case start="1419241497669" stop="1419241497670" status="canceled">
            <name>groupNameFieldOneSymbol</name>
            <failure>
                <message>RuntimeException: java.lang.NullPointerException</message>
                <stack-trace>java.lang.RuntimeException: java.lang.NullPointerException
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:161)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:429)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
Caused by: java.lang.NullPointerException
	at khppp.excel.utils.ExcelReader.getCellData(ExcelReader.java:69)
	at khppp.excel.utils.ExcelReader.getRowContains(ExcelReader.java:84)
	at khppp.core.CoreTest.testData(CoreTest.java:57)
	at khppp.tests.BaseCase.testData(BaseCase.java:54)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:135)
	... 23 more
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="feature" value="Groups"/>
            </labels>
        </test-case>
        <test-case start="1419241497670" stop="1419241499041" status="passed">
            <name>validAddGroupTabName</name>
            <steps>
                <step start="1419241497670" stop="1419241498158" status="passed">
                    <name>navigateToGroups</name>
                    <title>When I navigate to groups</title>
                    <attachments/>
                    <steps/>
                </step>
                <step start="1419241498158" stop="1419241498608" status="passed">
                    <name>navigateToGroups</name>
                    <title>When I navigate to groups</title>
                    <attachments/>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <labels>
                <label name="feature" value="Groups"/>
            </labels>
        </test-case>
        <test-case start="1419241499043" stop="1419241500251" status="passed">
            <name>verifyButtonsDisplayed</name>
            <steps>
                <step start="1419241499043" stop="1419241499472" status="passed">
                    <name>navigateToGroups</name>
                    <title>When I navigate to groups</title>
                    <attachments/>
                    <steps/>
                </step>
                <step start="1419241499472" stop="1419241499866" status="passed">
                    <name>navigateToGroups</name>
                    <title>When I navigate to groups</title>
                    <attachments/>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <labels>
                <label name="feature" value="Groups"/>
            </labels>
        </test-case>
        <test-case start="1419241500253" stop="1419241501468" status="passed">
            <name>verifyGroupNameField</name>
            <steps>
                <step start="1419241500253" stop="1419241500740" status="passed">
                    <name>navigateToGroups</name>
                    <title>When I navigate to groups</title>
                    <attachments/>
                    <steps/>
                </step>
                <step start="1419241500740" stop="1419241501158" status="passed">
                    <name>navigateToGroups</name>
                    <title>When I navigate to groups</title>
                    <attachments/>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <labels>
                <label name="feature" value="Groups"/>
            </labels>
        </test-case>
        <test-case start="1419241501470" stop="1419241503155" status="passed">
            <name>verifyLabels</name>
            <steps>
                <step start="1419241501470" stop="1419241501907" status="passed">
                    <name>navigateToGroups</name>
                    <title>When I navigate to groups</title>
                    <attachments/>
                    <steps/>
                </step>
                <step start="1419241501907" stop="1419241502653" status="passed">
                    <name>navigateToGroups</name>
                    <title>When I navigate to groups</title>
                    <attachments/>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <labels>
                <label name="feature" value="Groups"/>
            </labels>
        </test-case>
        <test-case start="1419241503158" stop="1419241503158" status="canceled">
            <name>creationOfMentee</name>
            <failure>
                <message>SkipException: The test was skipped for some reason</message>
                <stack-trace>org.testng.SkipException: The test was skipped for some reason
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:73)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1887)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1879)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1157)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels/>
        </test-case>
        <test-case start="1419241503159" stop="1419241503159" status="canceled">
            <name>displayGroupWithMentee</name>
            <failure>
                <message>SkipException: The test was skipped for some reason</message>
                <stack-trace>org.testng.SkipException: The test was skipped for some reason
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:73)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1887)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1879)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1157)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="feature" value="Groups"/>
            </labels>
        </test-case>
        <test-case start="1419241503159" stop="1419241503159" status="canceled">
            <name>menteeInAvailableMenteesField</name>
            <failure>
                <message>SkipException: The test was skipped for some reason</message>
                <stack-trace>org.testng.SkipException: The test was skipped for some reason
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:73)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1887)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1879)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1157)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="feature" value="Groups"/>
            </labels>
        </test-case>
        <test-case start="1419241503160" stop="1419241503160" status="canceled">
            <name>createGroupWithMentee</name>
            <failure>
                <message>SkipException: The test was skipped for some reason</message>
                <stack-trace>org.testng.SkipException: The test was skipped for some reason
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:73)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1887)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1879)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1157)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="feature" value="Groups"/>
            </labels>
        </test-case>
        <test-case start="1419241503161" stop="1419241503161" status="canceled">
            <name>removeMenteeFromGroup</name>
            <failure>
                <message>SkipException: The test was skipped for some reason</message>
                <stack-trace>org.testng.SkipException: The test was skipped for some reason
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:73)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1887)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1879)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1157)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="feature" value="Groups"/>
            </labels>
        </test-case>
        <test-case start="1419241513000" stop="1419241514959" status="passed">
            <name>preConditions[[LabManager_LabManager, LabManager111]]</name>
            <steps>
                <step start="1419241513232" stop="1419241514959" status="passed">
                    <name>login</name>
                    <title>When I login as LabManager_LabManager and LabManager111</title>
                    <attachments/>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <labels/>
        </test-case>
        <test-case start="1419241515052" stop="1419241515052" status="canceled">
            <name>addNewEmptySubgroup</name>
            <failure>
                <message>RuntimeException: java.lang.NullPointerException</message>
                <stack-trace>java.lang.RuntimeException: java.lang.NullPointerException
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:161)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:429)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
Caused by: java.lang.NullPointerException
	at khppp.excel.utils.ExcelReader.getCellData(ExcelReader.java:69)
	at khppp.excel.utils.ExcelReader.getRowData(ExcelReader.java:57)
	at khppp.excel.utils.ExcelReader.getTableArray(ExcelReader.java:45)
	at khppp.core.CoreTest.testData(CoreTest.java:58)
	at khppp.tests.BaseCase.testData(BaseCase.java:54)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:135)
	... 23 more
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="feature" value="Subgroup"/>
            </labels>
        </test-case>
        <test-case start="1419241515092" stop="1419241515092" status="canceled">
            <name>addNewSubgroup</name>
            <failure>
                <message>RuntimeException: java.lang.NullPointerException</message>
                <stack-trace>java.lang.RuntimeException: java.lang.NullPointerException
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:161)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:429)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
Caused by: java.lang.NullPointerException
	at khppp.excel.utils.ExcelReader.getCellData(ExcelReader.java:69)
	at khppp.excel.utils.ExcelReader.getRowData(ExcelReader.java:57)
	at khppp.excel.utils.ExcelReader.getTableArray(ExcelReader.java:45)
	at khppp.core.CoreTest.testData(CoreTest.java:58)
	at khppp.tests.BaseCase.testData(BaseCase.java:54)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:135)
	... 23 more
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="feature" value="Subgroup"/>
            </labels>
        </test-case>
        <test-case start="1419241521385" stop="1419241523227" status="passed">
            <name>preConditionsMentee[[Mentee_Mentee, Mentee111]]</name>
            <steps>
                <step start="1419241521608" stop="1419241523227" status="passed">
                    <name>login</name>
                    <title>When I login as Mentee_Mentee and Mentee111</title>
                    <attachments/>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <labels/>
        </test-case>
        <test-case start="1419241523258" stop="1419241545697" status="passed">
            <name>preConditionsMentor[[Mentor_Mentor, Mentor111]]</name>
            <steps>
                <step start="1419241526008" stop="1419241545695" status="passed">
                    <name>login</name>
                    <title>When I login as Mentor_Mentor and Mentor111</title>
                    <attachments/>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <labels/>
        </test-case>
        <test-case start="1419241545700" stop="1419241578079" status="broken">
            <name>changeStatusByMentee</name>
            <failure>
                <message>TimeoutException: Timed out after 30 seconds waiting for visibility of element located by By.xpath: .//*[@id='0']/td[1]
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=34.0.5, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webdriver.remote.sessionid=bf3fe967-a367-4ba0-8363-c25053aef59d, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: bf3fe967-a367-4ba0-8363-c25053aef59d</message>
                <stack-trace>org.openqa.selenium.TimeoutException: Timed out after 30 seconds waiting for visibility of element located by By.xpath: .//*[@id='0']/td[1]
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=34.0.5, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webdriver.remote.sessionid=bf3fe967-a367-4ba0-8363-c25053aef59d, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: bf3fe967-a367-4ba0-8363-c25053aef59d
	at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:79)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:228)
	at khppp.application.components.Component.waitFor(Component.java:73)
	at khppp.application.components.Component.waitFor(Component.java:83)
	at khppp.application.components.PersonalJournalPage.clickTaskName(PersonalJournalPage.java:19)
	at khppp.application.steps.StatusSteps.clickTaskNameStep(StatusSteps.java:21)
	at khppp.tests.StatusTest.changeStatusByMentee(StatusTest.java:64)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
Caused by: org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"xpath","selector":".//*[@id='0']/td[1]"}
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.44.0', revision: '76d78cf', time: '2014-10-23 20:02:37'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: driver.version: unknown
Command duration or timeout: 10.50 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=34.0.5, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webdriver.remote.sessionid=bf3fe967-a367-4ba0-8363-c25053aef59d, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: bf3fe967-a367-4ba0-8363-c25053aef59d
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:408)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:204)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:156)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:599)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:352)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:449)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:357)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:344)
	at org.openqa.selenium.support.ui.ExpectedConditions.findElement(ExpectedConditions.java:730)
	at org.openqa.selenium.support.ui.ExpectedConditions.access$000(ExpectedConditions.java:39)
	at org.openqa.selenium.support.ui.ExpectedConditions$4.apply(ExpectedConditions.java:130)
	at org.openqa.selenium.support.ui.ExpectedConditions$4.apply(ExpectedConditions.java:126)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:208)
	... 32 more
Caused by: org.openqa.selenium.remote.ScreenshotException: Screen shot has been taken
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: driver.version: RemoteWebDriver
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:136)
	... 42 more
Caused by: org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"xpath","selector":".//*[@id='0']/td[1]"}
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.44.0', revision: '76d78cf', time: '2014-10-23 20:02:37'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: driver.version: unknown
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: driver.version: unknown
	at &lt;anonymous class&gt;.FirefoxDriver.prototype.findElementInternal_(file:///C:/Users/YEHOR_~1/AppData/Local/Temp/anonymous1182964102041632494webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:9641:26)
	at &lt;anonymous class&gt;.fxdriver.Timer.prototype.setTimeout/&lt;.notify(file:///C:/Users/YEHOR_~1/AppData/Local/Temp/anonymous1182964102041632494webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:548:5)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels/>
        </test-case>
        <test-case start="1419241578109" stop="1419241578110" status="canceled">
            <name>changeStatusByMentor</name>
            <failure>
                <message>RuntimeException: java.lang.NullPointerException</message>
                <stack-trace>java.lang.RuntimeException: java.lang.NullPointerException
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:161)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:429)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
Caused by: java.lang.NullPointerException
	at khppp.excel.utils.ExcelReader.getCellData(ExcelReader.java:69)
	at khppp.excel.utils.ExcelReader.getRowData(ExcelReader.java:57)
	at khppp.excel.utils.ExcelReader.getTableArray(ExcelReader.java:45)
	at khppp.core.CoreTest.testData(CoreTest.java:58)
	at khppp.tests.BaseCase.testData(BaseCase.java:54)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:135)
	... 23 more
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="feature" value="Status"/>
            </labels>
        </test-case>
        <test-case start="1419241589019" stop="1419241590526" status="passed">
            <name>preConditions[[LabManager_LabManager, LabManager111, , , , , ]]</name>
            <steps>
                <step start="1419241589256" stop="1419241590526" status="passed">
                    <name>login</name>
                    <title>When I login as LabManager_LabManager and LabManager111</title>
                    <attachments/>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <labels/>
        </test-case>
        <test-case start="1419241590556" stop="1419241591772" status="failed">
            <name>validWithoutSubgroupTitle</name>
            <failure>
                <message>AssertionError: 
Expected: is &lt;true&gt;
     but: was &lt;false&gt;</message>
                <stack-trace>java.lang.AssertionError: 
Expected: is &lt;true&gt;
     but: was &lt;false&gt;
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
	at khppp.tests.GroupJournalTests.validWithoutSubgroupTitle(GroupJournalTests.java:117)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
</stack-trace>
            </failure>
            <steps>
                <step start="1419241590556" stop="1419241590989" status="passed">
                    <name>navigateToGroups</name>
                    <title>When I navigate to groups</title>
                </step>
            </steps>
            <attachments/>
            <labels>
                <label name="feature" value="GroupJournal"/>
            </labels>
        </test-case>
        <test-case start="1419241591773" stop="1419241593078" status="passed">
            <name>workOfBackButton</name>
            <steps>
                <step start="1419241591773" stop="1419241592143" status="passed">
                    <name>navigateToGroups</name>
                    <title>When I navigate to groups</title>
                    <attachments/>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <labels>
                <label name="feature" value="GroupJournal"/>
            </labels>
        </test-case>
        <test-case start="1419241593113" stop="1419241595547" status="passed">
            <name>creationOfMentee[[LabManager_LabManager, LabManager111, Anton, Antonov, MENTEE, , ]]</name>
            <steps>
                <step start="1419241593113" stop="1419241593546" status="passed">
                    <name>navigateTo</name>
                    <title>When I navigate to Users</title>
                    <attachments/>
                    <steps/>
                </step>
                <step start="1419241593546" stop="1419241595547" status="passed">
                    <name>addUserDifferentRoles</name>
                    <title>When I create users with different roles Anton, Antonov</title>
                    <attachments/>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <labels/>
        </test-case>
        <test-case start="1419241595550" stop="1419241627690" status="broken">
            <name>correctAddSubgroupBtnName</name>
            <failure>
                <message>TimeoutException: Timed out after 30 seconds waiting for visibility of element located by By.xpath: //div[@class='submenu']/a[text()='Add Subgroup']
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=34.0.5, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webdriver.remote.sessionid=dd32a7a8-bfdb-4842-9d30-47ff41b73d7a, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: dd32a7a8-bfdb-4842-9d30-47ff41b73d7a</message>
                <stack-trace>org.openqa.selenium.TimeoutException: Timed out after 30 seconds waiting for visibility of element located by By.xpath: //div[@class='submenu']/a[text()='Add Subgroup']
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=34.0.5, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webdriver.remote.sessionid=dd32a7a8-bfdb-4842-9d30-47ff41b73d7a, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: dd32a7a8-bfdb-4842-9d30-47ff41b73d7a
	at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:79)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:228)
	at khppp.application.components.Component.waitFor(Component.java:73)
	at khppp.application.components.Component.waitFor(Component.java:83)
	at khppp.application.components.GroupJournalPage.isBtnAddSubgroupDisplayed(GroupJournalPage.java:31)
	at khppp.application.steps.GroupJournalSteps.isAddSubgroupDisplayed(GroupJournalSteps.java:45)
	at khppp.tests.GroupJournalTests.correctAddSubgroupBtnName(GroupJournalTests.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
Caused by: org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"xpath","selector":"//div[@class='submenu']/a[text()='Add Subgroup']"}
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.44.0', revision: '76d78cf', time: '2014-10-23 20:02:37'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: driver.version: unknown
Command duration or timeout: 10.37 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=34.0.5, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webdriver.remote.sessionid=dd32a7a8-bfdb-4842-9d30-47ff41b73d7a, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: dd32a7a8-bfdb-4842-9d30-47ff41b73d7a
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:408)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:204)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:156)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:599)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:352)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:449)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:357)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:344)
	at org.openqa.selenium.support.ui.ExpectedConditions.findElement(ExpectedConditions.java:730)
	at org.openqa.selenium.support.ui.ExpectedConditions.access$000(ExpectedConditions.java:39)
	at org.openqa.selenium.support.ui.ExpectedConditions$4.apply(ExpectedConditions.java:130)
	at org.openqa.selenium.support.ui.ExpectedConditions$4.apply(ExpectedConditions.java:126)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:208)
	... 32 more
Caused by: org.openqa.selenium.remote.ScreenshotException: Screen shot has been taken
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: driver.version: RemoteWebDriver
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:136)
	... 42 more
Caused by: org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"xpath","selector":"//div[@class='submenu']/a[text()='Add Subgroup']"}
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.44.0', revision: '76d78cf', time: '2014-10-23 20:02:37'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: driver.version: unknown
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: driver.version: unknown
	at &lt;anonymous class&gt;.FirefoxDriver.prototype.findElementInternal_(file:///C:/Users/YEHOR_~1/AppData/Local/Temp/anonymous8143357805961477093webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:9641:26)
	at &lt;anonymous class&gt;.fxdriver.Timer.prototype.setTimeout/&lt;.notify(file:///C:/Users/YEHOR_~1/AppData/Local/Temp/anonymous8143357805961477093webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:548:5)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="feature" value="GroupJournal"/>
            </labels>
        </test-case>
        <test-case start="1419241627691" stop="1419241660421" status="broken">
            <name>correctAssignBtnName</name>
            <failure>
                <message>TimeoutException: Timed out after 30 seconds waiting for visibility of element located by By.xpath: //a[@class='btn btn-success'][2]
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=34.0.5, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webdriver.remote.sessionid=dd32a7a8-bfdb-4842-9d30-47ff41b73d7a, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: dd32a7a8-bfdb-4842-9d30-47ff41b73d7a</message>
                <stack-trace>org.openqa.selenium.TimeoutException: Timed out after 30 seconds waiting for visibility of element located by By.xpath: //a[@class='btn btn-success'][2]
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=34.0.5, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webdriver.remote.sessionid=dd32a7a8-bfdb-4842-9d30-47ff41b73d7a, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: dd32a7a8-bfdb-4842-9d30-47ff41b73d7a
	at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:79)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:228)
	at khppp.application.components.Component.waitFor(Component.java:73)
	at khppp.application.components.Component.waitFor(Component.java:83)
	at khppp.application.components.GroupJournalPage.isBtnAssignDisplayed(GroupJournalPage.java:35)
	at khppp.application.steps.GroupJournalSteps.isAssignDisplayed(GroupJournalSteps.java:49)
	at khppp.tests.GroupJournalTests.correctAssignBtnName(GroupJournalTests.java:91)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
Caused by: org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"xpath","selector":"//a[@class='btn btn-success'][2]"}
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.44.0', revision: '76d78cf', time: '2014-10-23 20:02:37'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: driver.version: unknown
Command duration or timeout: 10.52 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=34.0.5, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webdriver.remote.sessionid=dd32a7a8-bfdb-4842-9d30-47ff41b73d7a, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: dd32a7a8-bfdb-4842-9d30-47ff41b73d7a
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:408)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:204)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:156)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:599)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:352)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:449)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:357)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:344)
	at org.openqa.selenium.support.ui.ExpectedConditions.findElement(ExpectedConditions.java:730)
	at org.openqa.selenium.support.ui.ExpectedConditions.access$000(ExpectedConditions.java:39)
	at org.openqa.selenium.support.ui.ExpectedConditions$4.apply(ExpectedConditions.java:130)
	at org.openqa.selenium.support.ui.ExpectedConditions$4.apply(ExpectedConditions.java:126)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:208)
	... 32 more
Caused by: org.openqa.selenium.remote.ScreenshotException: Screen shot has been taken
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: driver.version: RemoteWebDriver
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:136)
	... 42 more
Caused by: org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"xpath","selector":"//a[@class='btn btn-success'][2]"}
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.44.0', revision: '76d78cf', time: '2014-10-23 20:02:37'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: driver.version: unknown
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: driver.version: unknown
	at &lt;anonymous class&gt;.FirefoxDriver.prototype.findElementInternal_(file:///C:/Users/YEHOR_~1/AppData/Local/Temp/anonymous8143357805961477093webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:9641:26)
	at &lt;anonymous class&gt;.fxdriver.Timer.prototype.setTimeout/&lt;.notify(file:///C:/Users/YEHOR_~1/AppData/Local/Temp/anonymous8143357805961477093webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:548:5)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="feature" value="GroupJournal"/>
            </labels>
        </test-case>
        <test-case start="1419241660422" stop="1419241692767" status="broken">
            <name>correctBackBtnName</name>
            <failure>
                <message>TimeoutException: Timed out after 30 seconds waiting for visibility of element located by By.xpath:  .//*[@id='group_journal_submit']/div[2]/a
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=34.0.5, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webdriver.remote.sessionid=dd32a7a8-bfdb-4842-9d30-47ff41b73d7a, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: dd32a7a8-bfdb-4842-9d30-47ff41b73d7a</message>
                <stack-trace>org.openqa.selenium.TimeoutException: Timed out after 30 seconds waiting for visibility of element located by By.xpath:  .//*[@id='group_journal_submit']/div[2]/a
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=34.0.5, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webdriver.remote.sessionid=dd32a7a8-bfdb-4842-9d30-47ff41b73d7a, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: dd32a7a8-bfdb-4842-9d30-47ff41b73d7a
	at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:79)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:228)
	at khppp.application.components.Component.waitFor(Component.java:73)
	at khppp.application.components.Component.waitFor(Component.java:83)
	at khppp.application.components.GroupJournalPage.isBtnBackDisplayed(GroupJournalPage.java:45)
	at khppp.application.steps.GroupJournalSteps.isBackDisplayed(GroupJournalSteps.java:57)
	at khppp.tests.GroupJournalTests.correctBackBtnName(GroupJournalTests.java:103)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:217)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:92)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
Caused by: org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"xpath","selector":" .//*[@id='group_journal_submit']/div[2]/a"}
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.44.0', revision: '76d78cf', time: '2014-10-23 20:02:37'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: driver.version: unknown
Command duration or timeout: 10.38 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=34.0.5, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webdriver.remote.sessionid=dd32a7a8-bfdb-4842-9d30-47ff41b73d7a, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: dd32a7a8-bfdb-4842-9d30-47ff41b73d7a
	at sun.reflect.GeneratedConstructorAccessor63.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:408)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:204)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:156)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:599)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:352)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:449)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:357)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:344)
	at org.openqa.selenium.support.ui.ExpectedConditions.findElement(ExpectedConditions.java:730)
	at org.openqa.selenium.support.ui.ExpectedConditions.access$000(ExpectedConditions.java:39)
	at org.openqa.selenium.support.ui.ExpectedConditions$4.apply(ExpectedConditions.java:130)
	at org.openqa.selenium.support.ui.ExpectedConditions$4.apply(ExpectedConditions.java:126)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:208)
	... 32 more
Caused by: org.openqa.selenium.remote.ScreenshotException: Screen shot has been taken
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: driver.version: RemoteWebDriver
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:136)
	... 42 more
Caused by: org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"xpath","selector":" .//*[@id='group_journal_submit']/div[2]/a"}
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.44.0', revision: '76d78cf', time: '2014-10-23 20:02:37'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: driver.version: unknown
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.44.0', revision: '76d78cf323ce037c5f92db6c1bba601c2ac43ad8', time: '2014-10-23 13:11:40'
System info: host: 'EPUAKHAW0662', ip: '192.168.56.1', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_25'
Driver info: driver.version: unknown
	at &lt;anonymous class&gt;.FirefoxDriver.prototype.findElementInternal_(file:///C:/Users/YEHOR_~1/AppData/Local/Temp/anonymous8143357805961477093webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:9641:26)
	at &lt;anonymous class&gt;.fxdriver.Timer.prototype.setTimeout/&lt;.notify(file:///C:/Users/YEHOR_~1/AppData/Local/Temp/anonymous8143357805961477093webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:548:5)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="feature" value="GroupJournal"/>
            </labels>
        </test-case>
        <test-case start="1419241692768" stop="1419241711232" status="passed">
            <name>correctExportBtnName</name>
            <steps/>
            <attachments/>
            <labels>
                <label name="feature" value="GroupJournal"/>
            </labels>
        </test-case>
        <test-case start="1419241711233" stop="1419241713074" status="passed">
            <name>correctGroupName</name>
            <steps/>
            <attachments/>
            <labels>
                <label name="feature" value="GroupJournal"/>
            </labels>
        </test-case>
        <test-case start="1419241713076" stop="1419241713151" status="passed">
            <name>correctPageName</name>
            <steps/>
            <attachments/>
            <labels>
                <label name="feature" value="GroupJournal"/>
            </labels>
        </test-case>
        <test-case start="1419241713152" stop="1419241713298" status="passed">
            <name>validGroupJournalTableHead</name>
            <steps/>
            <attachments/>
            <labels>
                <label name="feature" value="GroupJournal"/>
            </labels>
        </test-case>
        <test-case start="1419241713300" stop="1419241714462" status="passed">
            <name>availableMentees</name>
            <steps>
                <step start="1419241713300" stop="1419241713701" status="passed">
                    <name>navigateToGroups</name>
                    <title>When I navigate to groups</title>
                    <attachments/>
                    <steps/>
                </step>
                <step start="1419241713701" stop="1419241714134" status="passed">
                    <name>navigateToGroups</name>
                    <title>When I navigate to groups</title>
                    <attachments/>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <labels>
                <label name="feature" value="Groups"/>
            </labels>
        </test-case>
        <test-case start="1419241714847" stop="1419241717869" status="passed">
            <name>createGroupWithMentee[[Anton Antonov,   ,   ,   ,   , My_New_Group, QA]]</name>
            <steps>
                <step start="1419241714847" stop="1419241715375" status="passed">
                    <name>navigateToGroups</name>
                    <title>When I navigate to groups</title>
                    <attachments/>
                    <steps/>
                </step>
                <step start="1419241715376" stop="1419241715749" status="passed">
                    <name>navigateToGroups</name>
                    <title>When I navigate to groups</title>
                    <attachments/>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <labels>
                <label name="feature" value="Groups"/>
            </labels>
        </test-case>
    </test-cases>
    <labels/>
</ns2:test-suite>
